WordPress/Classes/ViewRelated/Reader/ReaderCardsStreamViewController.swift

```diff
@@ -25,6 +31,28 @@ class ReaderCardsStreamViewController: ReaderStreamViewController {
         }
     }
 
+    // MARK: - Sync
+
+    override func fetch(for topic: ReaderAbstractTopic, success: @escaping ((Int, Bool) -> Void), failure: @escaping ((Error?) -> Void)) {
+        cardsService.fetch(page: 1, success: success, failure: failure)
```

Comment:
Since this is resetting the data wouldn't we want to reset `currentPage` here?

Reply:
Yes! Done in 03ba076a452547ccd09dd4dcdd75c7304f10b278